content/400-pulse/200-getting-started.mdx

```diff
@@ -35,9 +35,10 @@ In the project environment of your choice, click the **Enable Pulse** button.
 
 The Pulse **Setup** screen requires you to:
 
-- provide your **Database connection string**
-- select a **Region** where Pulse should be hosted
-- decide whether you want to use the **Automatic setup** for [**Database replication**](/pulse/concepts#logical-replication) (only available on paid plans)
+1. provide your **Database connection string**
+1. select a **Region** where Pulse should be hosted
+1. decide whether you want to use the **Automatic setup** for **Database replication** (only available on paid plans)
```

Comment:
The link was removed. Should it be added back?

Reply:
```suggestion
1. decide whether you want to use the **Automatic setup** for [**Database replication**](/pulse/database-setup/general-database-instructions#enable-logical-replication) (only available on paid plans)
```